packages/boa/tools/check-dependence.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const { run, PLATFORM } = require('./utils');
+const version = process.version.split('.')[0];
+
+if (parseInt(version.substr(1)) < 12) {
+  throw new TypeError(`require Node.js >= v12.0.0, but ${process.version} found`);
+}
+
+let cmds = [ 'rm', 'make', 'tar' ];
+
+if (PLATFORM === 'linux') {
+  cmds = cmds.concat([ 'wget' ]);
```

Comment:
```suggestion
  cmds.push('wget');
```

And we can declare the `cmds` in constant.